daemon/cmd/ipam.go

```diff
@@ -338,7 +338,6 @@ func (d *Daemon) bootstrapIPAM() {
 	// Then, we will calculate the IPv4 or IPv6 alloc prefix based on the IPv6
 	// or IPv4 alloc prefix, respectively, retrieved by k8s node annotations.
 	bootstrapStats.ipam.Start()
```

Comment:
I think this should be removed (as well as the corresponding `bootstrapStats.ipam.End(true)` at line 363) since the functionality is moved to `(*Daemon).startIPAM()` by this PR.

Reply:
Done.